lib/peep.rb

```diff
@@ -0,0 +1,46 @@
+require "pg"
+require "time"
+
+class Peeps
+  def self.view
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    peeps = connection.exec("SELECT * FROM peeps")
+    peeps.map do |peep|
+      { :message => peep["message"] }
+    end
+  end
+  def self.add(message, date = Time.now)
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    connection.exec("INSERT INTO peeps (message, date, time) VALUES('#{message}', '#{date}', '#{date}')")
+  end
+  def self.date
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    peeps = connection.exec("SELECT * FROM peeps")
+    peeps.map do |peep|
+      { :date => peep["date"] }
+    end
+  end
+  def self.time
+    if ENV["ENVIRONMENT"] == "test"
+      connection = PG.connect(dbname: "chitter_test")
+    else
+      connection = PG.connect(dbname: "chitter")
+    end
+    peeps = connection.exec("SELECT * FROM peeps")
+    peeps.map do |peep|
+      { :time => peep["time"] }
+    end
+  end
+end
```

Comment:
You've done a good job of demonstrating some key learning objectives here (writing to and reading from a database).  There is some repetition, however, so the next step would probably be to remove that (rather than add more features).  How could you go about doing that and why is it important?